doc/create_isurf.txt

```diff
@@ -0,0 +1,162 @@
+"SPARTA WWW Site"_sws - "SPARTA Documentation"_sd - "SPARTA Commands"_sc :c
+
+:link(sws,http://sparta.sandia.gov)
+:link(sd,Manual.html)
+:link(sc,Section_commands.html#comm)
+
+:line
+
+create_isurf command :h3
+
+[Syntax:]
+
+create_isurf group-ID ablateID thresh mode ... :pre
+
+group-ID = grid ID for which grid cells to perform calculation on :ulb,l
+ablateID = ID of a "fix ablate"_fix_ablate.html command :l
+thresh = threshold for corner values used to generate implicit surfaces, value > 0.0 and < 255.0 :l
+mode = {inout} or {ave} = {inout} marks cells either in or out. {ave} smooths values by averaging expected corner points values based on intersections between the explicit surface and cell edge :l
+:ule
+
+[Examples:]
+
+create_isurf     all fablate 40.0 inout
+create_isurf     subset fablate 100.0 ave
+
+[Description:]
+
+This command converts the geometry of explicit surfaces read from a
+specified file into implicit surfaces. In SPARTA, a "surface" is a
+collection of surface elements that represent the surface of one or more
+physical objects which will be embedded in the global simulation box.
+Explicit surfaces are read from a previous "read_surf" command and must
+be "distributed". See the "read_surf"_read_surf.html command to read
+explicit surfaces from a different kind of file.  See the "How to
+6.13"_Section_howto.html#howto_13 section of the manual for an
+explantion of explicit versus implicit surfaces as well as distributed
+versus non-distributed storage.
+
+Explicit surface elements are triangles in 3d or line segments in 2d.  Surface
+elements for each physical object are required to be a complete,
+connected set that tile the entire surface of the object.  The surface
+elements in a single surface file must represent a "watertight" surface.
+For a 2d simulation this means that every point is part of exactly 2
+line segments.  For a 3d simulation it means that every triangle edge is
+part of exactly 2 triangles.  Exceptions to these rules allow for
+triangle edges (in 3d) that lie entirely in a global face of the
+simulation box, or for line points (in 2d) that are on a global edge of
+the simulation box.  This can be the case after clipping, which allows
+for use of watertight surface object (e.g. a sphere) that is only
+partially inside the simulation box, but which when clipped to the box
+becomes non-watertight, e.g. half of a sphere. Generated implicit surfaces will
+always be watertight.
+
+While not currently implemented, implicit surface elements can have a 
```

Comment:
Not sure what this means. Can you please clarify or remove.

Reply:
I removed the statement in the most recent commit